pkgs/tools/inputmethods/mouse-actions/default.nix

```
@@ -0,0 +1,24 @@
+{ lib, fetchFromGitHub, rustPlatform, pkg-config, virtualgl, xorg, libevdev }:
+rustPlatform.buildRustPackage rec {
+  pname = "mouse-actions";
+  version = "0.4.4";
+
+  src = fetchFromGitHub {
+    owner = "jersou";
+    repo = "mouse-actions";
+    rev = "v0.4.4";
+    hash = "sha256-02E4HrKIoBV3qZPVH6Tjz9Bv/mh5C8amO1Ilmd+YO5g=";
+  };
+
+  cargoHash = "sha256-GRClGC6+3J/YW5jVh7iAb2idD6/PycG854XbBIkfB/c=";
+  buildInputs = [ xorg.libX11 xorg.libXi xorg.libXtst libevdev ];
+  nativeBuildInputs = [ pkg-config virtualgl ];
```

```suggestion

  buildInputs = [
    libX11
    libXi
    libXtst
    libevdev
  ];

  nativeBuildInputs = [
    pkg-config
    virtualgl
  ];
```
Using separate lines makes it easier to see when new dependencies are added with git.